WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -1726,6 +1726,148 @@ function OptionsPrivate.OpenTriggerTemplate(data, targetId)
+  local lastType, miniGroup
+  for i, prop in ipairs(sortedProps) do
+    if prop.type == "header" then
+      local heading = AceGUI:Create("Heading")
+      heading:SetText(prop.name)
+      heading:SetRelativeWidth(1)
+      heading.label:SetFontObject(GameFontNormalSmall2)
+      frame.scrollList:AddChild(heading)
+    else
+      if ((prop.type == "mini" or prop.type == "marker") and prop.type ~= lastType)
+      then
+        miniGroup = AceGUI:Create("SimpleGroup")
+        miniGroup:SetLayout("Flow")
+        miniGroup:SetAutoAdjustHeight(true)
+        miniGroup:SetRelativeWidth(1)
+        frame.scrollList:AddChild(miniGroup)
+      end
+      local button = AceGUI:Create("WeakAurasSnippetButton")
+      local propIndex = prop.triggerNum > 0 and ("%s"):format(prop.triggerNum) or ""
+      local propPrefix = prop.triggerNum > 0 and ("%%%s."):format(propIndex) or "%"
+      if prop.type == "marker" then
+        button:SetTitle(prop.desc)
+      else
+        button:SetTitle(string.format("|cFFFFCC00%s|r%s", propPrefix, prop.name))
+      end
+      if prop.type == "mini" or prop.type == "marker" then
+        button:SetRelativeWidth((1/prop.n) - 1e-10)
+      else
+        button:SetRelativeWidth(1)
+      end
+      button.title:SetFontObject(GameFontNormal)
+      button.frame:SetHeight(28)
+
+      -- Modify highlight textures hover and push and remove normal texture
+      button.ntex:SetTexture(nil)
+      button.htex:SetTexture(nil)
+      button.ptex:SetTexture(nil)
+      button.htex:SetAtlas("Options_List_Hover")
+      button.htex:SetVertexColor(1, 1, 1, 1)
+      button.ptex:SetAtlas("Options_List_Active")
+
+      -- Set Tooltip
+      if prop.type ~= "marker" then
+        button.frame:SetScript("OnEnter", function(frame)
+          local tooltip = GameTooltip
+          tooltip:SetWidth(300)
+          tooltip:SetOwner(frame, "ANCHOR_RIGHT")
+          tooltip:ClearLines()
+          tooltip:AddLine(("%s%s"):format(propPrefix, prop.name))
+          tooltip:AddLine(prop.desc, 1, 1, 1, true)
+          tooltip:AddLine("\n")
+          tooltip:AddLine(
+            prop.triggerNum > 0
+            and L["The trigger number is optional. When no trigger number is specified, the trigger selected via dynamic information will be used."]
+            or L["By default this shows the information from the trigger selected via dynamic information. The information from a specific trigger can be shown via e.g. %2.p."],
+            0.8, 0.8, 0.8,
+            true)
+          tooltip:Show()
+          frame.obj:Fire("OnEnter")
+        end)
+      else
+        button.frame:SetScript("OnEnter", nil)
+      end
+
+      -- Insert dynamic text property on click
+      button:SetCallback("OnClick", function()
+        local insertProp
+        if prop.type == "marker" then
+          insertProp = prop.name
+        else
+          insertProp = prop.name == "%" and "%%" or ("%%{%s}"):format(prop.name)
```

Comment:
while %{variable} is the most correct, I think we should still insert without {}, that just looks better and is frequently the better choice.